includes/Themes/Fonts.php

```diff
@@ -100,6 +105,7 @@ protected static function get_theme_fonts() {
 				'bold-stamp-slab'                => array(
 					'label'   => __( 'Bold stamp & slab', 'wp-module-onboarding-data' ),
 					'matches' => 'yith-wonder/styles/05-red-shades',
+					'slugs'   => array( 'changa-one', 'roboto-slab' ),
```

Comment:
I see that you've used `'blocks'->'core/heading'->'typography'->'fontFamily'` elsewhere but here it is `'typography'->'fontFamily'` is there a specific reason?